felix/bpf/polprog/pol_prog_builder_test.go

```diff
@@ -66,11 +66,11 @@ func TestPolicySanityCheck(t *testing.T) {
 				}},
```

Comment:
Could do with a kick-the-tires test of the new function, maybe write a kitchen-sink rule, parse the resulting JSON into an object and compare with a canned one?